tests/index.js

```diff
@@ -1,40 +1,49 @@
 const dotenv = require('dotenv')
 const sinon = require('sinon')
+const tap = require('tap')
 const loadConfiguration = require('../index')
 
-exports.loadConfiguration = {
-  setUp: function (done) {
+tap.test('loadConfiguration', (module) => {
+  let configStub
+
+  module.beforeEach((done) => {
     this._env = process.env
     process.env = {}
     done()
```

Comment:
Is the `done()` call necessary if all the things are synchronous?

Reply:
yes